fast_forward/interactions_to_itp.py

```diff
@@ -0,0 +1,28 @@
+
+from vermouth.gmx.itp import write_molecule_itp
+from vermouth.file_writer import deferred_open
+
+def itp_writer(molname, block, interactions, command_used):
```

Comment:
I would have the interactions fitter directly update the block; you can replace the full interactions dict one type at a time